test/python/test_onedim.py

```diff
@@ -863,9 +863,9 @@ def check_save_restore(self, f):
         # pytest.approx is used as equality for floats cannot be guaranteed for loaded
         # HDF5 files if they were created on a different OS and/or architecture
         assert list(f.grid) == pytest.approx(list(self.sim.grid))
-        assert list(f.T) == pytest.approx(list(self.sim.T), rel=1e-5)
+        assert list(f.T) == pytest.approx(list(self.sim.T), rel=1e-4)
         k = self.gas.species_index('H2')
-        assert list(f.X[k, :]) == pytest.approx(list(self.sim.X[k, :]), rel=1e-5)
+        assert list(f.X[k, :]) == pytest.approx(list(self.sim.X[k, :]), rel=1e-4)
```

Comment:
Rather than relaxing the test tolerances, I think the "reference" files should be replaced with ones based on this PR.